### PR TITLE
Added colour_filter function and test

### PR DESCRIPTION
**Pull request by Sukriti1312**

### CODE REVIEW

File: pymagine/colour_filters.py [hunk 1]
```diff
@@ -4,6 +4,9 @@
 Implementation of colour_filters function in the pymagine package.
 """
 
+import matplotlib.pyplot as plt
+import numpy as np
+
```
Comment (trevorkwan) [inline pymagine/colour_filters.py:9]:
```suggestion
import cv2
```

File: pymagine/colour_filters.py [hunk 2]
```diff
@@ -25,3 +28,110 @@ def colour_filters(image, tone="sepia"):
...
+      transformed_red = (np.where(transformed_red > 255, 255, transformed_red)).astype(int)
+      transformed_green = (np.where(transformed_green > 255, 255, transformed_green)).astype(int)
+      transformed_blue = (np.where(transformed_blue > 255, 255, transformed_blue)).astype(int)
+      
+      n = image[:,:,0].shape
+      
+      final_image_array = np.dstack((np.reshape(transformed_red, n), np.reshape(transformed_green, n), np.reshape(transformed_blue, n)))
+      final_image = plt.imshow(final_image_array)
+  
+  return final_image
```
Comment (trevorkwan) [inline pymagine/colour_filters.py]:
```suggestion
  cv2.imwrite("colour_filters.jpeg", final_image)
```

File: pymagine/colour_filters.py [hunk 3]
```diff
@@ -4,6 +4,9 @@
 Implementation of colour_filters function in the pymagine package.
 """
 
+import matplotlib.pyplot as plt
+import numpy as np
+
 def colour_filters(image, tone="sepia"):
```
Comment (trevorkwan) [inline pymagine/colour_filters.py:10]:
```suggestion

def colour_filters(image, tone="sepia"):
```

**Review by trevorkwan — CHANGES_REQUESTED** [review]

File: pymagine/colour_filters.py (hunk 2 above)
Comment (Sukriti1312) [reply pymagine/colour_filters.py]: cv2.imwrite will not work with the plt.imshow object saved in `final_image`. I will have to edit some more portions of the file to save the image

File: tests/test_colour_filters.py [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+import os
+import pytest
+
+from pymagine import colour_filters as cf
+
+fname = os.path.join(os.path.dirname(__file__), '../tests/imgs/coronado_beach.jpeg')
+bad_ftype = 'imgs/coronado_beach.csv'
+url_fname = 'https://upload.wikimedia.org/wikipedia/commons/thumb/c/c3/Python-logo-notext.svg'
+
+def test_inputs():
```
Comment (katieb1) [inline tests/test_colour_filters.py:11]: Remember to add docstrings for the test function
```suggestion
def test_inputs():
"""
Applies tests to the colour_filters function to ensure proper usage.
"""
```

**Review by katieb1 — CHANGES_REQUESTED** [review]

**Review by katieb1 — APPROVED** [review]

**Review by katieb1 — APPROVED** [review]

**Review by trevorkwan — APPROVED** [review]

**Review by trevorkwan — APPROVED** [review]

**Pull request merged**